src/workerd/api/sql-test-tail.js

```diff
@@ -50,34 +45,37 @@ export default {
         resolveFn = resolve;
       })
     );
+    const topLevelSpanId = event.event.spanId;
 
     return (event) => {
       try {
-        const rpcMethodName = Array.isArray(event.event.info)
```

Comment:
This code is impossible to understand or maintain, this should not have been merged